.github/scripts/discord.sh

```diff
@@ -23,7 +23,11 @@ echo "[Webhook]: Preparing git related variables...";
 AUTHOR_NAME="$(git log -1 "$GITHUB_SHA" --pretty="%aN")"
 COMMITTER_NAME="$(git log -1 "$GITHUB_SHA" --pretty="%cN")"
 COMMIT_SUBJECT="$(git log -1 "$GITHUB_SHA" --pretty="%s")"
-COMMIT_MESSAGE="$(git log -1 "$GITHUB_SHA" --pretty="%b" | sed -E ':a;N;$!ba;s/\r{0,1}\n/\\n/g; s/"/\\"/g')"
+COMMIT_MESSAGE="$(git log -1 "$GITHUB_SHA" --pretty="%b")"
+#Replace newlines with literal \n
+COMMIT_MESSAGE=$(echo "$COMMIT_MESSAGE" | sed -E ':a;N;$!ba;s/\r{0,1}\n/\\n/g;')
```

Comment:
Although it was already part of the old expression, as `-E` is provided to `sed` you can use `?` instead of `{0,1}`:

```suggestion
COMMIT_MESSAGE=$(echo "$COMMIT_MESSAGE" | sed -E ':a;N;$!ba;s/\r?\n/\\n/g;')
```

And to also not fail when (old) MacOS line endings are used one can further modify the matching pattern:

```suggestion
COMMIT_MESSAGE=$(echo "$COMMIT_MESSAGE" | sed -E ':a;N;$!ba;s/\r?\n|\n?\r/\\n/g;')
```

But I don't view either of these changes as required.